### ISSUE TITLE
Check for accessibility issues

### ISSUE THREAD

**Issue by genderquery** [genderquery/averylarsen.com#7]
There aren't currently any accessibility issues reported by https://wave.webaim.org, but an automated check at build time would be preferable instead of manual review.

While the site may be accessible on a technical level, further research should be done on accessibility best practices.
